ch4/sec1/4-1-1.tex

```diff
@@ -12,11 +12,77 @@
 
 \setexercise{4}{1}{1}
 \begin{exercise}
-  
+Show that functors $F : C \to D$ and $G : D \to C$ and bijections $D(Fc, d) \cong C(c, Gd)$ for each $c \in C$ and $d \in D$ define an adjunction if and only if these bijections induce a bijection between commutative squares (4.1.4). That is, prove Lemma 4.1.3.
 \end{exercise}
 
 \begin{proof}
-  
+
+Let $C$ and $D$ be categories and let $F : C \to D , G : D \to C$ be functors.
+
+Suppose that we are given, for any $c\in C, d \in D$, a bijection \[ \varphi_{c,d}: D(Fc, d) \cong C(c, Gd) \]
+Denote its inverse by $\psi_{c,d}$. We will omit the subscripts if they are understood.
+
+Recall that this collection of bijections forms an adjunction (i.e. is natural in $C$ and $D$) if and only if the following identities hold:
+\begin{equation} \label{adj1}
```

Comment:
Ah, that's something I'd not thought about. These labels will be global in the entire collection of exercises, so I think we should have a convention: any label names should begin with the exercise number, so this would become `\label{4-1-1-adj1}`. It's more to type, but will avoid any conflicts between different solutions.